source/glossary.rst

```diff
@@ -227,43 +234,45 @@ Glossary of Terms
    MP3
      A patented audio compression algorithm which uses a form of :term:`lossy`
      data compression. It is de-facto standard of digital audio compression for
-     music. MP3 files generally end with a .mp3 extension.
+     music. MP3 files generally end with a :file:`.mp3` extension.
 
    AAC
-     Advanced Audio Coding (AAC) is a patented audio compression algorithm which
-     uses a form of :term:`lossy` data compression. Designed to be the successor
-     to :term:`MP3`, AAC generally achieves better sound quality at similar bit
-     rates. While the .aac extension is sometimes used for AAC-encoded files,
-     they are typically saved with an .m4a file extension.
+   Advanced Audio Coding 
+     A patented audio compression algorithm which uses a form of :term:`lossy`
+     data compression. Designed to be the successor to :term:`MP3`, AAC
+     generally achieves better sound quality at similar bit rates. While the
+     .aac extension is sometimes used for AAC-encoded files, they are typically
+     saved with an :file:`.m4a` file extension.
 
    FLAC
-     Free Lossless Audio Compression (FLAC), a patent-free audio compression
-     similar to :term:`MP3` but :term:`lossless` (i.e. there is no loss in audio
+   Free Lossless Audio Compression
+     A patent-free audio compression similar to :term:`MP3` but
+     :term:`lossless` (i.e. there is no loss in audio
      quality when used). FLAC files generally end with a .flac extension.
 
    ALAC
      Apple Lossless Audio Codec (ALAC) is a :term:`lossless` audio file format
      used by iTunes and other Apple products. ALAC files use the same MP4
-     container format as AAC and use the same file extensions (m4a, mp4). Mixxx
-     and many other programs are not compatible with ALAC. It is recommended
-     to convert ALAC files to :term:`FLAC` for use in Mixxx.
+     container format as AAC and use the same file extensions (:file:`m4a`,
```

Comment:
let#s put 'Apple Lossless Audio Codec' on a new line like you did for the other abbreviations